Add usage to README.md (#13)
ESSS/deps: README.md

`deps inv codegen -j 16 --jobs-unordered` will run `inv codegen` on the projects and its dependencies
with full-paralelization, without respecting any particular order.

Usage
=====

> Program to list dependencies of a project, or to execute a command for each dependency:

Reviewer: ```suggestion
Program to list development dependencies of `conda-devenv` projects, or to execute a command for each dependency:
```

    deps [OPTIONS] [COMMAND]...

> To list dependency projects, one per line (if "-p directory" is omitted, it will use the current, or will find the first ancestor directory containing an `environment.devenv.yml` file):

Reviewer: ```suggestion
To list project dependencies of the project in the current directory, one per line:

    deps 
  
It is possible to pass `-p directory` to find the first ancestor of `directory` containing an `environment.devenv.yml` file:
```
